rfcs/044-sierra-series/README.md

```diff
@@ -0,0 +1,503 @@
+              "id": "issn",
+              "label": "ISSN",
+              "type": "IdentifierType"
+            },
+            "value": "1064-3745",
+            "type": "Identifier"
+          }
+        ],
+        "title": "Methods in Molecular Biology, Methods and Protocols",
+        "type": "Series"
+      }
+    ],
+    "type": "Series"
+  },
+  {   // 773
+    "title": "Springer eBooks",
+    "type": "Work"
+  }
+]
+```
+- [3001878](https://search.wellcomelibrary.org/iii/encore/record/C__Rb3001878?lang=eng&suite=cobalt&marcData=Y)
+  Only 830 with no 490/440 or 773
+```yaml
+"partOf": [
+    {
+        "title": "Adamson Collection Wellcome Library", // subfield $a concatenated with $p
+        "type": "Series"
+    }
+]
+```
+- [1110225](https://search.wellcomelibrary.org/iii/encore/record/C__Rb1110225?lang=eng&suite=cobalt&marcData=Y)
+Series statement in 440 with only the title
+```yaml
+"partOf": [
+    {
+        "title": "Monographs on inorganic and physical chemistry",
+        "type": "Series"
+    }
+]
+```
+- [1204561](https://search.wellcomelibrary.org/iii/encore/record/C__Rb1204561?lang=eng&suite=cobalt&marcData=Y) 440 with id
+```yaml
+"partOf": [
+  {
+    "title": "Brill's studies in intellectual history; v. 13",
+    "partOf": [
+      {
+        "id": "abcdefgh",
+        "identifiers": [
+          {
+            "identifierType": {
+              "id": "issn",
+              "label": "ISSN",
+              "type": "IdentifierType"
+            },
+            "value": "0920-8607",
+            "type": "Identifier"
+          }
+        ],
+        "title": "Brill's studies in intellectual history",
+        "type": "Series"
+      }
+    ],
+    "type": "Series"
+  }
+]
+```
+- [2301867](https://search.wellcomelibrary.org/iii/encore/record/C__Rb2301867?lang=eng&suite=cobalt&marcData=Y)
+  773 not overlapping with series and no id.
+```yaml
+"partOf": [
+    {
+        "title": "Eighteenth Century collections online",
+        "type": "Work"
+    }
+]
+```
+- [1186777](https://search.wellcomelibrary.org/iii/encore/record/C__Rb1186777?lang=eng&suite=cobalt&marcData=Y)
+  773 with an id in subfield $w which links to another Work in our library (see example below)
+```yaml
+"partOf": [
+   {
+    "title": "Basil Hood. Photograph album; page 9",
+    "partOf": [
+      {
+        "id": "abcdefgh",
+        "identifiers": [
+          {
+            // identifier from $w
+            "identifierType": {
+              "id": "???",
+              "label": "???",
+              "type": "IdentifierType"
+            },
+            "value": "(Wcat)9175i",
+            "type": "Identifier"
+          }
+        ],
+        "title": "Basil Hood. Photograph album",
+        "type": "Work"
+      }
+    ],
+    "type": "Work"
+  }
+]
+```
+- [1172977](https://search.wellcomelibrary.org/iii/encore/record/C__Rb1172977?lang=eng&suite=cobalt&marcData=Y)
+  774 with ids (related to above 773)
+```yaml
+"parts": [
+  {
```

Comment:
In the existing API, I think parts objects must have an id. However, unless we do treat individual pages or volumes as objects in their own right, I don't think this interface change is avoidable.